tests/Functional/E2ETest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Functional;
+
+use Symfony\Component\Panther\PantherTestCase;
+use Sulu\Component\HttpKernel\SuluKernel;
+
+class E2ETest extends PantherTestCase
+{
+    public function testMyApp()
+    {
+        $client = static::createPantherClient();
+        $crawler = $client->request('GET', '/admin');
+
+        $this->assertContains('SULU', $crawler->filter('title')->html());
+
+        $client->waitFor('form');
+
+        $crawler->selectButton('button:last-child')->form([
+            '_username' => 'admin',
```

Comment:
currently there exists no names on the inputs which make e2e tests really hard /cc @danrot @chirimoya

Reply:
Feel free to add them :wink: